apps/browse/views.py

```diff
@@ -240,8 +240,7 @@ def __init__(self, request, base, category, key, default):
                                                     default)
 
     def filter_featured(self):
-        # Never fear this will get & with manual order and the base.
-        return Addon.objects.all()
+        return self.base_queryset.all()
 
     def order_featured(self, filter):
```

Comment:
Seems you're right, well spotted! I'll also remove the now useless `_filter` helper method.